EpiResponsivePicture.Tests/ImageTransformationHelperTests.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Collections.Generic;
+using System.Web.Mvc;
+using EPiServer;
+using EPiServer.Core;
+using EPiServer.ServiceLocation;
+using EPiServer.Web.Routing;
+using Moq;
+using NUnit.Framework;
+using Forte.EpiResponsivePicture.ResizedImage;
+
+namespace Forte.EpiResponsivePicture.Tests
+{
+    [TestFixture]
+    public class ImageTransformationHelperTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+            var contentLoaderMock = new Mock<IContentLoader>();
+
+            var urlResolverMock = new Mock<IUrlResolver>();
+            urlResolverMock
+                .Setup(r => r.GetUrl(It.IsAny<ContentReference>(), It.IsAny<string>(),
+                    It.IsAny<UrlResolverArguments>())).Returns("/images/foo.jpg");
+
+            ServiceLocator.SetLocator(new DummyServiceLocator(new Dictionary<Type, object>
+            {
+                {typeof(IContentLoader), contentLoaderMock.Object},
+                {typeof(IUrlResolver), urlResolverMock.Object}
+            }));
+        }
+
+        [Test]
+        public void TestResizedImageUrl()
+        {
+            HtmlHelper html = null;
+
+            var urlWithoutFormat = html.ResizedImageUrl(new ContentReference(123), 500).ToString();
```

Comment:
SRP applies also for tests. Actually these should be two separate test cases (applies also to second test)